nbqa/__main__.py

```diff
@@ -298,7 +342,7 @@ def _run_command(
     command: str,
     tmpdirname: str,
     cmd_args: List[str],
-    arg: Path,
+    args: List[Path],
```

Comment:
minor hint: a list is slower/takes memory compared to tuples

Reply:
thanks! will change

Reply:
Given the size of `args` is going to be really small (since it holds the directories or notebooks to run nbqa) does it really matter whether its a tuple or list?  I am thinking in this case its just better to leave as it is since `args` is generated by list comprehension. Maybe using `args: Iterable[Path]`  would make for a better abstraction.

Reply:
I agree for the size it's not substantial to take advantage of my earlier suggestion.

Reply:
In PyMC3 this list can potentially have 80+ notebooks if you run nbqa via pre-commit, so I made the change to tuple anyway, thanks!